books/spiders/products.py

```diff
@@ -0,0 +1,52 @@
+from typing import Any
+
+import scrapy
+from scrapy.http import Response
+
+
+RATING_MAPPER = {
+    "One": 1,
+    "Two": 2,
+    "Three": 3,
+    "Four": 4,
+    "Five": 5,
+}
```

Comment:
you could use word2number package  to make it more automated https://pypi.org/project/word2number/